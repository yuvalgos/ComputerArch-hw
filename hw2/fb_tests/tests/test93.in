r 0x770d8
w 0x770b8
r 0x770c8
w 0x77104
w 0x770cc
r 0x77148
r 0x77114
w 0x770fc
w 0x77118
w 0x7710c
w 0x77100
r 0x7712c
r 0x77124
w 0x770d0
w 0x77100
r 0x770fc
w 0x77118
w 0x77148
w 0x77158
r 0x77148
r 0x77154
w 0x77100
r 0x77108
r 0x7713c
w 0x77124
r 0x770e4
r 0x77120
r 0x77114
r 0x770b8
w 0x770b8
r 0x77130
r 0x770dc
r 0x770c8
w 0x7711c
w 0x77148
w 0x770f8
r 0x77150
w 0x77154
w 0x77128
r 0x770d0
r 0x77154
