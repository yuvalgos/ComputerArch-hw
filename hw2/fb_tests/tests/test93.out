L1miss=0.805 L2miss=0.667 AccTimeAvg=129.244
